pkgs/by-name/em/emulationstation/package.nix

```diff
@@ -1,31 +1,32 @@
-{ lib
-, SDL2
-, alsa-lib
-, boost
-, cmake
-, curl
-, fetchFromGitHub
-, freeimage
-, freetype
-, libGL
-, libGLU
-, libvlc
-, pkg-config
-, rapidjson
-, stdenv
+{
+  lib,
+  SDL2,
+  alsa-lib,
+  boost,
+  callPackage,
+  cmake,
+  curl,
+  freeimage,
+  freetype,
+  libGL,
+  libGLU,
+  libvlc,
+  pkg-config,
+  rapidjson,
+  stdenv,
+  # Configurable options
+  sources ? callPackage ./sources.nix { },
 }:
 
 stdenv.mkDerivation (finalAttrs: {
```

Comment:
Perhaps `finalAttrs` is no longer needed?

Reply:
Yes, but I see no necessity of removing it.

Reply:
Unused attributes should be removed

Reply:
This is not an attribute, but an argument for `mkDerivation`.